DigitalLearningSolutions.Data/DataServices/CourseDataService.cs

```diff
@@ -357,7 +408,7 @@ public IEnumerable<DelegateCourseInfo> GetDelegateCoursesInfo(int delegateId)
             return connection.Query<DelegateCourseInfo>(
                 $@"{selectDelegateCourseInfoQuery}
                     WHERE pr.CandidateID = @delegateId
-                        AND ap.ArchivedDate IS NULL
+                        --AND ap.ArchivedDate IS NULL

```

Comment:
This is actually something that slipped past my notice due to how I relied on the Rider syntax highlighting. I'll just check whether this one really wants to be removed (the one below does because it searches by progressId directly).

Reply:
It goes to the ViewDelegateController, so I'll remove it.